src/modules/accounting.py

```diff
@@ -0,0 +1,251 @@
+            {"internalType":"uint256","name":"withdrawalsReserveAmount","type":"uint256"},
+            {"internalType":"uint256[]","name":"requestIdToFinalizeUpTo","type":"uint256[]"},
+            {"internalType":"uint256[]","name":"finalizationShareRates","type":"uint256[]"}
+        ) external;
+        """
+        beacon_validators_count, beacon_validators_balance = self._get_beacon_validators_stats(slot, block_hash)
+        logger.info({
+            'msg': 'Fetch lido validators.',
+            'beacon_validators_count': beacon_validators_count,
+            'beacon_validators_balance': beacon_validators_balance,
+        })
+        ACCOUNTING_ACTIVE_VALIDATORS.set(beacon_validators_count)
+        ACCOUNTING_VALIDATORS_BALANCE.set(beacon_validators_balance)
+
+        (
+            stacking_module_ids,
+            no_operators,
+            exited_validators,
+        ) = self._get_new_exited_validators(slot, block_hash)
+        logger.info({
+            'msg': 'Fetch Lido exited validators.',
+            'stacking_module_ids': stacking_module_ids,
+            'no_operators': no_operators,
+            'exited_validators': exited_validators,
+        })
+        ACCOUNTING_EXITED_VALIDATORS.set(len(exited_validators))
+
+        wc_balance = self._get_wc_balance(block_hash)
+        logger.info({'msg': 'Fetch wc balance.', 'value': wc_balance})
+        WC_BALANCE.set(wc_balance)
+
+        wei_to_reserve = self._get_wei_to_reserve(block_hash)
+        logger.info({'msg': 'Calculate ether to reserve.', 'value': wei_to_reserve})
+        WEI_TO_RESERVE.set(wei_to_reserve)
+
+        (
+            last_requests_id_to_finalize,
+            finalization_shares_amount,
+        ) = self._get_requests_finalization_report(block_hash)
+        logger.info({
+            'msg': 'Get withdrawals queue stats.',
+            'last_requests_id_to_finalize': last_requests_id_to_finalize,
+            'finalization_shares_amount': finalization_shares_amount,
+        })
+        if last_requests_id_to_finalize:
+            LAST_FINALIZED_WITHDRAWAL_REQUEST.set(last_requests_id_to_finalize[-1])
+
+        transaction = contracts.oracle.functions.handleCommitteeMemberReport((
+            int(slot / self.slots_per_epoch),
+            beacon_validators_count,
+            beacon_validators_balance,
+            stacking_module_ids,
+            no_operators,
+            exited_validators,
+            wc_balance,
+            wei_to_reserve,
+            last_requests_id_to_finalize,
+            finalization_shares_amount,
+        ))
+
+        return transaction
+
+    def _get_beacon_validators_stats(self, slot: SlotNumber, block_hash: HexBytes) -> Tuple[int, int]:
+        # Should return balance in gwei!
+        lido_validators = get_lido_validators(self._w3, block_hash, self._beacon_chain_client, slot)
+        return len(lido_validators), sum(int(validator['validator']['balance']) for validator in lido_validators)
```

Comment:
Is it ok sum validators' balances and cast value into (int). I'm afraid that that string can lose some affective balance, like 165.3242 ETH.